src/types/editor.ts

```diff
@@ -63,6 +63,14 @@ export type Resource = {
 	type: string;
 };
 
+export type ChipResource = {
+	id: string;
+	label: string;
+	email: string;
+	avatarIcon: 'BriefcaseOutline' | 'BuildingOutline';
+	avatarBackground: 'transparent';
+	avatarColor: 'gray0';
```

Comment:
a constant would make the code more sustainable in case of future default values changes

Reply:
This is a type, any suggestion on how to manage it?